### PR TITLE
Fix non-GPU build

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: src/core/backend_context.cc [hunk 1]
```diff
@@ -1241,6 +1254,7 @@ BackendInputCollector::FlushPendingCopyKernel(
     const TRITONSERVER_MemoryType tensor_memory_type,
     const int64_t tensor_memory_type_id)
 {
+  LOG_ERROR << "before launching gather kernel";
```
Comment (CoderHam) [inline src/core/backend_context.cc:1257]: Did you mean to add these temporarily for logging. I think they should be removed.

File: src/core/backend_context.cc [hunk 2]
```diff
@@ -1254,6 +1268,7 @@ BackendInputCollector::FlushPendingCopyKernel(
         tensor_buffer, tensor_buffer_byte_size, tensor_memory_type,
         tensor_memory_type_id);
     cuda_copy = status.IsOk();
+    LOG_ERROR << "gather kernel launched with status: " << status.AsString();
```
Comment (CoderHam) [inline src/core/backend_context.cc:1271]: Same here.

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Review by CoderHam — APPROVED** [review]

**Comment by deadeyegoodwin** [comment]
@GuanLuo  It seems like that extra logging for every request could have had an impact of perf. What do you think?

**Comment by GuanLuo** [comment]
That is debug logging for checking PTX.. Removed from this PR

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**